.spelling

```diff
@@ -97,577 +55,120 @@
 802.3cd
 8325-23C
 8325-48Y8C
-8325s
-83xx
-8x
-99-slingshot-network
-9f
-a.k.a.
-A47_2.40_02_23_2021.signed.flash
-A48_2.40_02_24_2021.signed.flash
-AAA.BBB.CCC.DDD
-accel
-access_token
-accfpga_nvidia_2.7.tar.gz
 ACLs
-acpid
-actionID
-additionalInventoryUrl
-addr
-admin.conf
-admin.conf.
-afix
-Agg
-ahead_
 airgapped
-AJ:MIN RM
-alertmanager
-alertmanager.cmn.shasta.dev.cray.com
-alertmanager.ncn
-alertmanager.shasta.dev.cray.com
-allowableDeviceStates
-allowed-vlan
 analog
 analyze
 analyzer
 Ansible
-ansible-0
-ansible-cfg
-ansible-verbosity
-ansible.cfg
 AOS-CX
-api
-api-gateway
-api-gw
-api-gw-service-nmn
-api.can.shasta.dev.cray.com
-api.can.SYSTEM_DOMAIN_NAME
-api.chn.shasta.dev.cray.com
-api.chn.SYSTEM_DOMAIN_NAME
-api.cmn.shasta.dev.cray.com
-api.cmn.SYSTEM_DOMAIN_NAME
-api.nmnlb
-api.nmnlb.shasta.dev.cray.com
-api.nmnlb.SYSETM_DOMAIN_NAME
-api.shasta.dev.cray.com
-apis
-app.kubernetes.io
-apparmor-profiles
-application-node-config-yaml
-application_node_config.yaml
-apply_csm_configuration.sh
 Arista
-Aristas
-arm.2020
-arptables
 artifact
 artifact's
 artifacts
 ArubaOS-CX
-Arubas
 ASes
 ASICs
-asn
-auth
-auth-type
-auth.cmn.shasta.dev.cray.com
-auth.shasta.dev.cray.com
-authority.crt
-authority.crt.
-autoinst
-autoyast2
-autoyast2-installation
-b8
 backdoor
 backend
 BardPeak
 Barebones
-barebonesImageTest
 base-64
 base-64-encoded
 base64
 base64-encoded
 Basecamp
 Basecamp's
-baseos
-batcherCheckInterval
-batchSize
-batchWindow
-bB
-bb28a35a-6cbc-4c30-84b0-6050314af76b
-Bdl
 behavior
 behaviors
-BestEffort
-bgp
-BGP-Mode
 BiCAN
-bind-utils
-biosdevname
 Bitnami
 blackhole
-blockedBy
-BlockRole
 BMCs
-boa-8f2fc013-7817-4fe2-8e6f-c2136a5e3bd1
-bond0
-bond0.can0
-bond0.cmn0
-bond0.hmn0
-bond0.nmn0
-bond1
-book.html
 bootscript
-bootstrap-ncn-bmc-pass
-bootstrap-ncn-bmc-user
-bootstrap_dhcp
-bootsys` to start the Kubernetes cluster. 
-boot_set_name
-bos
 BOSv2
-BOS_TEMPLATE
-bpdu-guard
-bs
-bsr-candidate
-bss
-buildenv-sidecar
-bundle.der
 Burstable
 busses
-c3
-ca.crt
-cabinets-yaml
-cabinets.yaml
-cable_info_disconnected
 cadvisor
-can-cidr
 CAN-to-CMN
-CAN.json
 Canu
 CANU-generated
-canu.yaml
-capmc
 CAs
-CASM-1885
-CASMINST-2015
-CASMINST-255
-CASMPET-338
 Cassini
-Catalog
 cataloged
-cc-1
-cd
-cdu
 CDUs
 CECs
-Center
 centers
 Ceph
-ceph-ansible
-ceph-common
-ceph-csi
-ceph-fs
-ceph-image
-ceph-mds
-ceph-mgr
-ceph-mon
-ceph-osd
-ceph-pool-quotas
-ceph-radosgw
-ceph-rgw-users
-ceph-storage
-Ceph-Volume
-ceph.conf
 Cephadm
 CephFS
-cephx
-Ceph_Orchestrator_Usage.md
-ceph_rbd
-ceph_rbd_
-cert-dir
-cert8.db
-cert8.db.bak
-cert9.db
-cert9.db.bak
-Certificate_authority
-cfg.json
-cfgsh
-cfs
 CFS-Batcher
 CFS-BOA
-cfs-default-ansible-cfg
-cfs-e8e48c2a-448f-4e6b-86fa-dae534b1702e-pnxmn
-cfs-state-reporter
-changeme
-Change_NCN_Image_Root_Password_and_SSH_Keys_on_PIT_Node
 channeled
-ChassisBMC
-ChassisBMCs
-ChassisSensors
 checkmark
-ChildVerificationFailed
-chn
```

Comment:
Should we have all of the capitalized network names in here?  NMN, CHN, CAN, CMN, HMN, NMNLB, HMNLB, NMN_RVR, HMN_RVR, NMN_MTN, HMN_MTN since these are acronyms rather than filenames, service names, etc.

Reply:
Yeah, I removed it since it was lowercase, but you're right that it would make sense to instead convert those to all caps. I'll make that update.